mlx/directives/attribute_sort_directive.py

```diff
@@ -54,5 +54,8 @@ def run(self):
         node['filter'] = r"\S+"
         node['sort'] = []
 
-        self.process_options(node, {'sort': [], 'filter': r"\S+"})
+        self.process_options(node,
+                             {'sort':   {'default': []},
+                              'filter': {'default': r"\S+"}
+                              })
```

Comment:
```suggestion:-2+0
                             {
                                 'sort':   {'default': []},
                                 'filter': {'default': r"\S+"},
                              })
```
I prefer this style of indentation

Reply:
See e5dc5dc and 5d5ba9c